msal/src/cache/java/com/microsoft/identity/client/BaseTokenCacheItem.java

```diff
@@ -31,13 +31,13 @@
 abstract class BaseTokenCacheItem {
 
     @SerializedName("client_id")
-    final String mClientId;
+    String mClientId;
```

Comment:
@weijjia You have to for the empty constructor. Plus, those values are only initialized if the server returns those values else they will be null.